src/app/container/info-tab/info-tab.component.html

```diff
@@ -51,12 +51,32 @@
               <a [href]="trsLinkCWL">
                 {{ tool?.tool_path }}
               </a>
+              <button
+                mat-icon-button
+                color="secondary"
+                ngxClipboard
+                [cbContent]="trsLinkCWL"
+                (cbOnSuccess)="isCopied1 = true"
```

Comment:
Do you need this? I see the pattern was already there, but I'm wondering if it has any purpose. Same comment for other 2 occurrences.

Reply:
I used another copy button on the accounts page as a template for this one, so I thought all of these parts were necessary in order for the button to work